client/my-sites/theme/main.jsx

```diff
@@ -9,6 +9,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import page from 'page';
+import { isPremium } from 'my-sites/themes/helpers';
```

Comment:
Not an external dependency :-) Plus we're also importing from that file [here](https://github.com/Automattic/wp-calypso/pull/5282/files#diff-f576b3c13daa17f9ccee514c4c46bc07R30) -- so that line should be changed to

```js
import { isPremium, getForumUrl } from 'my-sites/themes/helpers';
```

(And https://github.com/Automattic/wp-calypso/pull/5282/files#diff-f576b3c13daa17f9ccee514c4c46bc07R177 needs to be changed accordingly.)